### ISSUE TITLE
Groupfolders crashes Nextcloud with OCP\Files\NotPermittedException: No create permission for folder

### ISSUE THREAD

**Issue by rabesocke** [nextcloud/groupfolders#1021]
I have a problem after moving to Nextcloud in Docker. Since this move I cannot activate `groupfolders` anymore.

The installation is possible, but when I create a group folder the whole Nextcloud crashes and logs the following error.

Steps to reproduce:

1. Install `groupfolders` from the App section
2. Go to system settings - Group folders
3. Create a group folder "Test" with all rights to group "admin" and "1 GB" limit.
4. Go to the base path

Installation information:

- nextcloud 19.0.1 (Docker image nextcloud:19)
- Groupfolders 7.0.0
- MariaDB 10.4.14-MariaDB-1:10.4.14+maria~focal (Docker image mariadb:10.4)

```
[index] Error: OCP\Files\NotPermittedException: No create permission for folder at <<closure>>

 0. <<closure>>
    OC\Files\Node\Folder->newFolder(7)
 1. /var/www/html/custom_apps/groupfolders/lib/Helper/LazyFolder.php line 72
    call_user_func_array([OC\Files\Node\Folder {},"newFolder"], [7])
 2. /var/www/html/custom_apps/groupfolders/lib/Helper/LazyFolder.php line 152
    OCA\GroupFolders\Helper\LazyFolder->__call("newFolder", [7])
 3. /var/www/html/custom_apps/groupfolders/lib/Mount/MountProvider.php line 185
    OCA\GroupFolders\Helper\LazyFolder->newFolder(7)
 4. /var/www/html/custom_apps/groupfolders/lib/Mount/MountProvider.php line 122
    OCA\GroupFolders\Mount\MountProvider->getFolder(7)
 5. /var/www/html/custom_apps/groupfolders/lib/Mount/MountProvider.php line 88
    OCA\GroupFolders\Mount\MountProvider->getMount(7, "/Christian/files/Test", 31, -3, null, OC\Files\Storage\StorageFactory {}, false, OC\User\User {})
 6. <<closure>>
    OCA\GroupFolders\Mount\MountProvider->OCA\GroupFolders\Mount\{closure}("*** sensitive parameters replaced ***")
 7. /var/www/html/custom_apps/groupfolders/lib/Mount/MountProvider.php line 86
    array_map(Closure {}, ["*** sensitive parameter replaced ***"])
 8. /var/www/html/lib/private/Files/Config/MountProviderCollection.php line 105
    OCA\GroupFolders\Mount\MountProvider->getMountsForUser(OC\User\User {}, OC\Files\Storage\StorageFactory {})
 9. /var/www/html/lib/private/Files/Filesystem.php line 451
    OC\Files\Config\MountProviderCollection->addMountForUser(OC\User\User {}, OC\Files\Mount\Manager {})
10. /var/www/html/lib/private/Files/Filesystem.php line 376
    OC\Files\Filesystem::initMountPoints("Christian")
11. /var/www/html/lib/private/legacy/OC_Util.php line 308
    OC\Files\Filesystem::init("Christian", "/Christian/files")
12. /var/www/html/lib/base.php line 1006
    OC_Util::setupFS()
13. /var/www/html/index.php line 37
    OC::handleRequest()
```

**Comment by pierreozoux** [comment]
Are you sure about your permissions at the filesystem level?
Did you try a repair or a filescan?

**Comment by rabesocke** [comment]
I tried to set the permissions several times from inside the container and on host.  I also uninstall and install the app several times and even delete the folder in the data directory. Every time I get the same error.

At the moment I ended up to set up a new installation from scratch with Docker and migrate all the necessary data and database tables. In this installation, groupfolders works out of the box without problems.

We can probably close this issue if you want, because I resolved the problem with this new installation and I haven't found any other discussion about this topic (so it seems, I'm the only one having this problem).
